monai/networks/nets/vnet.py

```diff
@@ -224,7 +224,8 @@ def __init__(
         in_channels: int = 1,
         out_channels: int = 1,
         act: tuple[str, dict] | str = ("elu", {"inplace": True}),
-        dropout_prob: float = 0.5,
```

Comment:
thanks for the PR, would be great to keep this argument backward compatible for a while -- when the user specifies `dropout_prob=` or relies on the default value, it can still work but generates warnings about the code changes. (docstring should be updated as well)

please see an example of handling the deprecated args here:
https://github.com/Project-MONAI/MONAI/blob/af0e0e9f757558d144b655c63afcea3a4e0a06f5/monai/transforms/post/array.py#L157-L178

Reply:
Addressed in 0e9240716839f4d8c3d71e76ffecd431aef37c7f